R/xdfutils.R

```diff
@@ -0,0 +1,42 @@
+#' extract levels from xdf data
+#'
+#' @param form R formula
+#' @param xdf path to xdf file
+#' @return names of levels - vector
+#' @export
+getXdfLevels <- function(form, xdf) {
+  factors <- rxSummary(form, data = xdf)$categorical
```

Comment:
Use standard argument names wherever possible. `formula` is a well used argument name and it will be nice to stick to it.